components/PageControl/src/MDCPageControl.m

```diff
@@ -193,6 +194,7 @@ - (void)setCurrentPage:(NSInteger)currentPage
     [CATransaction setDisableActions:YES];
     [_indicators[previousPage] setHidden:NO];
     [CATransaction commit];
+    [self sendActionsForControlEvents:UIControlEventValueChanged];
```

Comment:
Hrm. Does UIButton dispatch when the user lifts their finger or after the animation completes?